modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/PrecisionAtN.java

```diff
@@ -117,24 +127,24 @@ public EvalQueryQuality evaluate(SearchHit[] hits, RatedQuery intent) {
 
         return new EvalQueryQuality(precision, unknownDocIds);
     }
-    
+
     public enum Rating {
-        RELEVANT, IRRELEVANT;
+        IRRELEVANT, RELEVANT;
```

Comment:
I switched this to IRRELEVANT(0), RELEVANT(1) because I found this more intuitive and it actually confused me quiet a bit when I got this wrong accidentally when working on the rest test.